pages/science/reactors.rst

```diff
@@ -351,11 +355,18 @@ to the reactors previously mentioned:
 
   .. math::
 
-     \dot m = \max(\dot m_0, 0.0)
+     \dot m = m g(t)
 
-  where :math:`\dot m_0` is either a constant value or a function of time. Note
-  that if :math:`\dot m_0 < 0`, the mass flow rate will be set to zero, since
-  reversal of the flow direction is not allowed.
+  where *m* is a mass flow coefficient and *g* is a function of time that is set
```

Comment:
Can you put `m` and `g` in math formatting? I'm also not sold on the symbol `m` for the coefficient. It would probably be good to use `\dot{m}_0` like in the class docstring. Can you specify how one sets this coefficient, like you've done for the function?

Reply:
Code changed, as have the class docstrings.